### PR TITLE
Altcoin presta 1.7

### PR DESCRIPTION
**Pull request by DarrenWestwood**

### CODE REVIEW

**Comment by shivaenigma** [comment]
Few issues
- The text style is differing between [BTC](https://screenshots.firefox.com/d4ufUQadzmvFOIIb/localhost) and [altcoin page](https://screenshots.firefox.com/7EveB9Q4TBVatMcl/localhost)
- Order number is not shown on altcoin page
- Back from altcoin page is not working

File: views/js/app.js [hunk 1]
```diff
@@ -16,48 +16,491 @@
...
+            $scope.copyshow = false;
+        }, 2000);
+    }
+    //Define supported altcoins
+    $scope.altcoins = {
+        "ETH": "Ethereum",
+        "LTC": "Litecoin"
+    };
 });
+
```
Comment (shivaenigma) [inline views/js/app.js]: Remove this

File: views/js/app.js [hunk 2]
```diff
@@ -16,48 +16,491 @@
...
     $scope.clock = totalTime;
+    $scope.copyshow = false;
+
+    //Create url when the order is received 
+    $scope.finish_order_url = function() {
+        return final_url;
+    }
+
+    //Decode the tracking url
+    $scope.alt_decode_url = function(url) {
```
Comment (shivaenigma) [inline views/js/app.js]: Please use decodeURI or decodeURIComponent. if it doesn't work... add reference to stackexchange thread

File: views/css/bootstrap-prestashop-ui-kit.css [hunk 3]
```diff
@@ -2,7 +2,7 @@
 /* not used */
 /* TODO change padding */
 @import url("https://fonts.googleapis.com/css?family=Open+Sans:300,400,500,600,700,800");
-@import url(jquery.pstagger.css);
```
Comment (shivaenigma) [inline views/css/bootstrap-prestashop-ui-kit.css]: Can you explain why we commented this ?

**Comment by shivaenigma** [comment]
- Remove unused views/css/cryptofont/cryptofont.css
- When changing tab from BTC to ETH highlight is not chaning

File: views/css/bootstrap-prestashop-ui-kit.css (hunk 3 above)
Comment (DarrenWestwood) [reply views/css/bootstrap-prestashop-ui-kit.css]: I have re-instated this as it is causing errors in the console, but not breaking functionality. 
` [8e8aa02](/blockonomics/prestashop-plugin/pull/58/commits/8e8aa02ec6478847ccb5449298f2cc78cf9198eb)`

**Comment by shivaenigma** [comment]
Refund details are showing empty even though flypme returned those fields
screenshot from 2019-03-01 20-14-25

File: views/templates/front/altcoin.tpl [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+ * @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ * International Registered Trademark & Property of Blockonomics
+ *}
+{extends $layout}
+
+{block name="content"}
+
+<div class="form" ng-app="blockonomics-invoice">
+  <script>var ajax_url = "{$ajax_url|escape:'htmlall':'UTF-8'}";</script>
+  <script>var get_uuid = "{$uuid|escape:'htmlall':'UTF-8'}";</script>
```
Comment (shivaenigma) [inline views/templates/front/altcoin.tpl]: Is this used ? Please remove if unused

File: views/js/app.js [hunk 5]
```diff
@@ -16,48 +16,491 @@
...
+        //Create a new altcoin order
+        create_order(getParameterByNameBlocko("altcoin"), getParameterByNameBlocko("amount"), getParameterByNameBlocko("address"), getParameterByNameBlocko("order_id"));
+    } 
+    else {
+        //Check the info for altcoin order
+        info_order(getParameterByNameBlocko("uuid"));
+    }
+
+    //Create url for refund page
+    $scope.alt_refund_url = function(uuid) {
```
Comment (shivaenigma) [inline views/js/app.js]: Remove unused function

File: views/js/app.js [hunk 6]
```diff
@@ -16,48 +16,491 @@
...
+                            });
+                            //Fetch the order id
+                            AltcoinAjax.get({
+                                'action': 'fetch_order_id',
+                                'address': address
+                            },function(order_id) {
+                                $scope.id_order = order_id.id;
+                            });
+                            $scope.altuuid = uuid;
+                            $scope.refundlink = $scope.alt_refund_url(uuid);
```
Comment (shivaenigma) [inline views/js/app.js]: Remove unused variable and function

File: views/templates/front/altcoin.tpl (hunk 4 above)
Comment (DarrenWestwood) [reply views/templates/front/altcoin.tpl]: Removed unused variables. 77145834a021608a34eb96d11ef4658140178d1e

File: views/js/app.js (hunk 6 above)
Comment (DarrenWestwood) [reply views/js/app.js]: Fix refund link variable name. fdc3751b98471674738297d7175f341f668b7b8c

**Pull request merged**